aewb_logger/aewb_logger_receiver.h

```diff
@@ -0,0 +1,34 @@
+#ifndef _AEWB_LOGGER_RECEIVER_H
+#define _AEWB_LOGGER_RECEIVER_H
+
+#include <sys/socket.h>
+#include <sys/types.h>
+#include <arpa/inet.h>
+#include <stdlib.h>
+#include "aewb_logger_types.h"
+#include "aewb_logger_receiver_types.h"
```

Comment:
either order includes from longest to shortest or the other way around, but you must decide!

Reply:
fixed